src/main/kotlin/dev/haarolean/gnutardbot/abilities/ForwardSuggestionsAbility.kt

```diff
@@ -0,0 +1,43 @@
+package dev.haarolean.gnutardbot.abilities
+
+import dev.haarolean.gnutardbot.TardBot
+import dev.haarolean.gnutardbot.util.MemberUtils
+import org.telegram.abilitybots.api.objects.MessageContext
+import org.telegram.telegrambots.meta.api.methods.ForwardMessage
+import org.telegram.telegrambots.meta.api.methods.groupadministration.GetChat
+import org.telegram.telegrambots.meta.api.objects.Message
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException
+
+class ForwardSuggestionsAbility(private val bot: TardBot) : AbilityHandler {
+    private val memberUtils: MemberUtils = (MemberUtils(bot))
+
+    override fun isApplicable(ctx: MessageContext): Boolean {
+        val update = ctx.update()
+        val message: Message = when {
+            update.hasMessage() -> update.message
+            update.hasEditedMessage() -> update.editedMessage
+            else -> return false
+        }
+        if (bot.isAdmin(message.from.id)) return false
```

Comment:
message.from.id is supposed to be a user chat, so it's always false as the bot can't be an admin of a private chat. This whole thing seems redundant